### ISSUE TITLE
flake8 support

### ISSUE THREAD

**Issue by gaborbernat** [tox-dev/tox-ini-fmt#34]
Add support for formatting flake8 sections.

**Comment by jugmac00** [comment]
All possible options for **flake8**: 🙈
https://flake8.pycqa.org/en/latest/user/options.html

**Comment by gaborbernat** [comment]
We don't have to support all out of box 🤔 maybe a few common ones just first and iterate on it.

**Comment by jugmac00** [comment]
I gave this a shot and it kinda works, but once again there is this "comment" topic.

While #42 is about deleted comments, when parsing one of my production `tox.ini` files with a `flake8` section I encountered the problem of inline comments. Allowed or not ( https://github.com/tox-dev/tox/issues/50 ), they work for third party tools.

```
[flake8]
ignore =
    W504, # line break after binary operator
quiet = 1
```
Whereas the comment in the`ConfigParser._read` function says inline comments get stripped, this one does not:
https://github.com/python/cpython/blob/96a60ae90c291d94c058c80351fa38b6d73eda92/Lib/configparser.py#L567

```
(Pdb++) parser[section_name]
<Section: flake8>
(Pdb++) parser[section_name].items()
ItemsView(<Section: flake8>)
(Pdb++) list(parser[section_name].items())
[('ignore', '\nW504, # line break after binary operator')]
```

So, before going down the rabbithole and debugging ConfigParser itself...

What's your take on the whole comment story?
